examples/Workflow/WorkflowWebApp/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should this be in the repo? (I was asked to remove it from samples in another repo.)

Reply:
Ok. So we are keeping this. I will resolve this conversation then.